packages/renderer/src/lib/featured/FeaturedExtensionDownload.svelte

```diff
@@ -71,6 +71,7 @@ async function installExtension(): Promise<void> {
 }
 </script>
 
+<ErrorMessage icon wrapMessage class="-top-[15px] -right-0 absolute" error={errorInstall}/>
```

Comment:
question (non-blocking): the classes seems a bit odd to me? compare to the one bellow you added a `-` before `right-0`, is there any reason?

Reply:
That must be a left-over from alignment experiments. Fixing the negative-zero value to a positive-zero :-)